src/ds/ds5/ds5-device.h

```diff
@@ -105,7 +95,6 @@ namespace librealsense
         std::shared_ptr<stream_interface> _color_stream;
 
         uint8_t _depth_device_idx;
-        uint16_t _pid;
```

Comment:
Consider removing it from l500 / sr300 other ?

Reply:
Done